_posts/en/newsletters/2023-12-20-newsletter.md

```diff
@@ -619,17 +622,18 @@ contract. This method is private to participants, requires less onchain
 space, and likely eliminates offchain rebalancing fees, thus improving
 revenue potential for forwarding nodes and reliability for LN payments.
 However, it requires a multiparty state contract, which is untested in
-production.  ZmnSCPxj suggests using [LN-Symmetry][topic eltoo] or
-duplex payment channels as a foundation, with duplex channels being more
-feasible despite their inability to remain open indefinitely.
+production.  ZmnSCPxj suggests building on [LN-Symmetry][topic eltoo]
+(which requires a consensus change) or duplex payment channels (whach
+can be implemented today), with duplex channels being more feasible
+today despite their inability to remain open indefinitely.
```

Comment:
"(which can be implemented today)" and "being more feasible today" feels a bit like a repetition